modules/transport.js

```diff
@@ -369,7 +348,7 @@ Transport.prototype.onUnconfirmedTransaction = function(
 	if (broadcast && !__private.broadcaster.maxRelays(transaction)) {
 		__private.broadcaster.enqueue(
 			{},
-			{ api: 'postTransactions', data: { transaction } }
+			{ api: 'postTransactions', data: { transactions: [transaction] } }
```

Comment:
Is there a way that we remove this inconsistency around the code? If there is an array of transactions then there is an array. Why we are having an `object` with a field `transactions` which is an array. We remove this issue in HTTP layer, if possible we should remove it from transport as well.

Reply:
Good point.

Reply:
@nazarhussain I thought about this a bit more and discussed it with @MaciejBaj.

It's true that the format `{fieldName: ...}` is not consistent with the HTTP API but it is consistent with all other RPC calls. For example the `postBlock` RPC function is in the form `{block: ...}`. In RPC functions the ` {transactions: [transaction] }` object is treated as a `query`; having it as an object with properties (instead of passing an array directly) gives more flexibility to extend it in the future.